4_Zadania_dodatkowe/8.Parallax/js/app.js

```diff
@@ -0,0 +1,46 @@
+$(document).ready(function () {
+
+    var oldMousePositionX = 0;
+    var oldMousePositionY = 0;
+
+    var currentMousePositionX = 0;
+    var currentMousePositionY = 0;
+
+    var mouseMoveX = 0;
+    var mouseMoveY = 0;
+
+    $('.element')
+        .each(function() {
+            $(this).css('z-index', $(this).data('z'))
+        })
+        .each(function() {
+            $(this).css('left', $(this).data('x'))
+        })
+        .each(function() {
+            $(this).css('top', $(this).data('y'))
+        });
```

Comment:
you don't have to write `each` method multiple times

you can do:
```
.each(function() {
    $(this)
        .css('z-index', $(this).data('z'))
        .css('left', $(this).data('x'))
        .css('top', $(this).data('y'));
})
```
or even
```
.each(function() {
    $(this).css({
        'z-index': ...,
        'left': ...,
        'top': ...,
    });
})
```

Reply:
Neat - like it :)